esp/src/src/nls/hu/hpcc.ts

```diff
@@ -468,7 +475,23 @@ export = {
     Log: "Log",
     LogAccessType: "Log hozzáférés típusa",
     LogDirectory: "Log könyvtár",
+    LogEventType: "Eseménynapló bejegyzés típus",
     LogFile: "Log fájl",
+    LogFilterComponentsFilterTooltip: "Csak a kiválasztott komponenshez/konténerhez tartozó eseménynapló bejegyzések",
+    LogFilterCustomColumnsTooltip: "A megadott eseménynapló oszlopok bevonása",
+    LogFilterEndDateTooltip: "Naplósorok felvétele az időtartomány végéig",
+    LogFilterEventTypeTooltip: "Csak a kiválasztott tipusú eseménynapló bejegyzásek",
+    LogFilterFormatTooltip: "Eseménynapló fájl formátum: CSV, JSON vagy XML",
+    LogFilterLineLimitTooltip: "Az eseménynapló-sorok száma, amelyek hozzáadásra kerülnek",
+    LogFilterLineStartFromTooltip: "Tartalmazza az eseménynapló-sorokat, amelyek ettől a sorszámtól kezdődnek",
+    LogFilterRelativeTimeRangeTooltip: "'A WU időpontját körülvevő időtartomány, +/- (másodpercben)'",
+    LogFilters: "Eseménynapló szűrők",
+    LogFilterSelectColumnModeTooltip: "Az eseménynapló fájlban szereplő  oszlopok megadása",
```

Comment:
Typo, extra space. The correct one is:
 "Az eseménynapló fájlban szereplő oszlopok megadása"